mods/tuxemon/maps/maple_bedroom.tmx

```diff
@@ -1,108 +1,72 @@
+  <image source="../gfx/tilesets/core_set pieces.png" width="496" height="800"/>
+ </tileset>
+ <layer id="1" name="Tile Layer 1" width="9" height="7">
   <data encoding="base64" compression="zlib">
-   eAFjYCANMAKVE8IwE8WBDEIYptYYyMCGlYHiIAySgwFs6kBielBMjFpkMwiZS6lamPmEaABISgmm
+   eAFjYmBgYCKAJYDyhLAuUM1QwwCSPQjU
   </data>
  </layer>
- <layer id="2" name="Tile Layer 2" width="11" height="9">
+ <layer id="2" name="Tile Layer 2" width="9" height="7">
   <data encoding="base64" compression="zlib">
-   eAFjYKAuYMZhXAxQPA5Njg+ND+OWABllUM5uIL0HJkGAPgyUP4KmphaNj4/biU8STW4qEj8RyE4C4mQkMXKYAFhXB3U=
+   eAFjYMAP+Jkg8kqMDAzKQAwC1UBcA2ZBCD2oGkegvBNUzSyg1GwgTgTiJIgynGQmUCYLpyymxA1MIQyR72gi9UB3NUDdBpMCAJGDB6A=
   </data>
  </layer>
- <layer id="3" name="Tile Layer 3" width="11" height="9">
+ <layer id="3" name="Tile Layer 3" width="9" height="7">
   <data encoding="base64" compression="zlib">
-   eAFjYKAPWAi0ZhGRVk0HqgNhUsFRoIZjpGoaROoBtvkD/Q==
+   eAFjYBg4sB9qdTaQzhkAZwAAw5QBlw==
   </data>
  </layer>
- <layer id="4" name="Above player" width="11" height="9">
+ <layer id="4" name="Above player" width="9" height="7">
   <data encoding="base64" compression="zlib">
-   eAFjYKA/2Aq0chsOa2vQxPcC+fvQxIYaVxLoYEIY5CcAdX0ERA==
+   eAFjYCAfRAO1xhDQngyUTyGgZqCkAQCPAX8=
   </data>
  </layer>
  <objectgroup color="#ff0000" id="5" name="Collision">
-  <object id="1" type="collision" x="16" y="32" width="144" height="16"/>
-  <object id="2" type="collision" x="160" y="48" width="16" height="80"/>
-  <object id="3" type="collision" x="16" y="128" width="144" height="16"/>
-  <object id="4" type="collision" x="0" y="48" width="16" height="80"/>
-  <object id="6" type="collision" x="112" y="64" width="32" height="16"/>
-  <object id="7" type="collision" x="112" y="48" width="16" height="16"/>
+  <object id="1" type="collision" x="0" y="16" width="144" height="16"/>
+  <object id="6" type="collision" x="96" y="48" width="32" height="16"/>
+  <object id="7" type="collision" x="64" y="32" width="48" height="16"/>
+  <object id="32" type="collision" x="16" y="32" width="32" height="16"/>
+  <object id="33" type="collision" x="0" y="80" width="16" height="32"/>
+  <object id="35" type="collision" x="64" y="96" width="32" height="16"/>
  </objectgroup>
  <objectgroup color="#ffff00" id="6" name="Events">
-  <object id="9" name="Go Downstairs" type="event" x="128" y="48" width="16" height="16">
+  <object id="9" name="Go Downstairs" type="event" x="112" y="32" width="16" height="16">
    <properties>
-    <property name="act10" value="transition_teleport maple_house_downstairs.tmx,4,2,0.3"/>
+    <property name="act10" value="transition_teleport maple_house.tmx,11,2,0.3"/>
     <property name="act20" value="player_face down"/>
     <property name="cond10" value="is player_at"/>
     <property name="cond20" value="is player_facing left"/>
    </properties>
   </object>
-  <object id="19" name="Create Maple" type="event" x="0" y="32" width="16" height="16">
```

Comment:
@Qiangong2 ok to remove this event?

Reply:
I don't believe I ever actually used the maple bedroom map.

I've used the player_bedroom map, but not maple.

Reply:
You're right, I forgot to remove it, I'm going to update.